src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -161,6 +161,13 @@ module sonic-port{
 					}
 				}
 
+				leaf dom_polling {
+
+					type string {
```

Comment:
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L127-L132
Can you use admin_mode from sonic-types.yang?

Reply:
@ganglyu - I have addressed this now.